torch/distributed/_tensor/ops/_math_ops.py

```diff
@@ -979,6 +978,7 @@ def layer_norm_bwd_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> OpStrategy
             )
         else:
             output_specs_list.append(None)
+            op_args_target_specs.append(None)
```

Comment:
Good suggestion! I've refactored accordingly. I also noticed both weight and bias share target input spec logic so I refactored that into a small nested function (`_add_target_input_spec`). 
At the moment weight and bias largely share output spec logic that could be refactored as well but I've refrained from implementing that because of the indication that we might change the weight spec construction.